docs/serving/services/deployment.md

```diff
@@ -42,5 +42,5 @@ metadata:
   namespace: knative-serving
 data:
 ...
-  progressDeadline: "10m"
+  progressDeadline: "600s"
```

Comment:
Why are you changing to seconds? 

The description above mentions 10 minutes so "10m" seems more natural. 

is the description mentioned 600 seconds then using "600s" then is more natural. 

using such a hi number seconds in an example comes with cognitive load. 

if the example needs to in seconds then a value of 120s or less is easier to reason about.

Reply:
👍 
I see no resaon for this change.

Reply:
but when i checked the actual deployed configmap yaml it is in minutes.
![Screenshot from 2021-02-23 11-29-59](https://user-images.githubusercontent.com/37874414/108807455-84fdc800-75ca-11eb-9e0a-6acd3b63d19d.png)

Reply:
Ah I see perhaps would be good idea to update the configMap embedded docs to indicate that a and m are supported and change 600s to 10m